worker/caasunitprovisioner/application_worker.go

```diff
@@ -125,6 +130,16 @@ func (aw *applicationWorker) loop() error {
 				return errors.Annotatef(err, "failed to start operator watcher for %q", aw.application)
 			}
 		}
+		if appDeploymentWatcher == nil {
+			appDeploymentWatcher, err = aw.serviceBroker.WatchService(aw.application)
```

Comment:
`appServiceWatcher`?

Reply:
The Watch() api is on the serviceBroker interface along with a few other methods. There's not many so no real benefit to split up.